components/org.wso2.carbon.identity.oauth.endpoint/src/main/java/org/wso2/carbon/identity/oauth/endpoint/authz/OAuth2AuthzEndpoint.java

```diff
@@ -3384,43 +3404,38 @@ private void associateAuthenticationHistory(SessionDataCacheEntry resultFromLogi
     }
 
     /**
-     * Returns the SessionContext associated with the cookie, if there is a one.
-     *
-     * @param cookie
+     * Returns the SessionContext associated with the cookie value, if there is a one.
+     * @param cookieValue String value of the cookie of commonAuthId.
      * @param loginTenantDomain Login tenant domain.
-     * @return the associate SessionContext or null.
+     * @return he associate SessionContext or null.
```

Comment:
was this change intended?

Reply:
Fixed ad3cb637ce2bd2485bc1b335e6f68be0d87ef70b